public/creators-landing/src/sections/main-section/index.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import {
+  Container,
+  GradientBackground,
+  H2,
+  PrimaryButton,
+  UserIcon,
+  YouTubeIcon,
+  PublicationIcon,
+  TwitchIcon,
+  SwoopBottom,
+  StyledInput
+} from '../../components'
+import { Heading, Text, Box, Anchor } from 'grommet'
+import locale from '../../locale/en'
+
+export const MainHome = () => {
+  return (
+    <GradientBackground align='center'>
+      <Container animation='fadeIn' role='main'>
+        <Box
+          direction='column'
+          pad='large'
+          margin={{ top: '80px', bottom: '60px' }}
+        >
+          <Heading
+            level='1'
+            color='white'
+            margin={{ vertical: 'small' }}
+            a11yTitle='Headline'
+          >
+            {locale.main.home.headline}
+          </Heading>
+          <H2 level='2' size='small' color='#E9E9F4' a11yTitle='Subtitle'>
+            {locale.main.home.subhead}
+          </H2>
+          <Heading level='3' size='small' color='#E9E9F4'>
+            {locale.main.home.examples.headline}
+          </Heading>
+          <Box direction='row-responsive' gap='24px'>
+            <Box direction='column'>
+              <Box direction='row' gap='small' margin={{ vertical: '8px' }}>
+                <UserIcon />
+                <Heading level='3' color='#E9E9F4' size='small' margin='0'>
+                  {locale.main.home.examples.website}
+                </Heading>
+              </Box>
+              <Box direction='row' gap='small' margin={{ vertical: '8px' }}>
+                <YouTubeIcon />
+                <Heading level='3' color='#E9E9F4' size='small' margin='0'>
+                  {locale.main.home.examples.youtube}
+                </Heading>
+              </Box>
+            </Box>
+            <Box direction='column'>
+              <Box direction='row' gap='small' margin={{ vertical: '8px' }}>
+                <PublicationIcon />
+                <Heading level='3' color='#E9E9F4' size='small' margin='0'>
+                  {locale.main.home.examples.publication}
+                </Heading>
+              </Box>
+              <Box direction='row' gap='small' margin={{ vertical: '8px' }}>
+                <TwitchIcon />
+                <Heading level='3' color='#E9E9F4' size='small' margin='0'>
+                  {locale.main.home.examples.Twitch}
+                </Heading>
+              </Box>
+            </Box>
+          </Box>
+          <Box direction='row' pad={{ vertical: '24px' }} width='100%'>
+            <PrimaryButton
+              label={locale.main.home.btn.signup}
+              a11yTitle='Sign up'
+            />
+          </Box>
+        </Box>
+      </Container>
+      <SwoopBottom />
```

Comment:
I was having problems where the swoop bottom was on top of the "Sign Up" button so I couldn't click on the bottom half when my web browser was like 75% full screen. Can you investigate?

Reply:
great spot. swoop was on top of bottom 1/6 of button. pushed down.